core/runtime/jsrealm.rs

```diff
@@ -197,7 +197,8 @@ impl JsRealmInner {
 
       let module_map =
         ctx.get_aligned_pointer_from_embedder_data(MODULE_MAP_SLOT_INDEX);
-      let _ = Box::from_raw(module_map as *mut Rc<ModuleMap>);
+      let module_map = Box::from_raw(module_map as *mut Rc<ModuleMap>);
```

Comment:
We are we putting an [Rc in a Box](https://www.youtube.com/watch?v=5Cx7jzq2Bx4)?

Reply:
@littledivy I missed that in the review. Was there a specific reason we use `Box<Rc<T>>`?

Reply:
Yeah I wonder this too. I can't think of a reason it would be necessary - to V8 it's just an opaque pointer, and
 `Rc::into_raw` + `Rc::from_raw` should be fine to use.

Reply:
`Rc::into_raw` returns a `*const T` but we want a `*const Rc<T>` to clone it. Another workaround is:

```rust
fn clone_rc_raw(raw: *const T) -> Rc<T> {
  unsafe { Rc::increment_strong_count(raw) };
  unsafe { Rc::from_raw(ptr) }
}
```

Reply:
https://github.com/denoland/deno_core/pull/778